### PR TITLE
[ID] Add "Offsets"

### PR DESCRIPTION
**Pull request by mousesalt**
Add Indonesia translation of
- [Universal offset](https://osu.ppy.sh/wiki/en/Universal_offset)
- [Local song offset](https://osu.ppy.sh/wiki/en/Glossary%2FLocal_song_offset)
- [How to use the Offset Wizard](https://osu.ppy.sh/wiki/en/Guides%2FHow_to_Use_the_Offset_Wizard)
- [How to set the offset on the correct beat](https://osu.ppy.sh/wiki/en/Guides%2FSetting_the_Offset_on_the_Correct_Beat)
- [Offset Wizard](https://osu.ppy.sh/wiki/en/Options%2FOffset_Wizard)

### CODE REVIEW

File: wiki/Glossary/Local_song_offset/id.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+stub: true
+---
+
+# Offset lokal
+
+*Untuk offset ketika membuat sebuah beatmap, kunjungi: [Offset](/wiki/Beatmapping/Offset)*
+
+**Offset lokal** adalah offset yang memindahkan tampilan [hit object](/wiki/Hit_object) dalam kaitannya dengan audio [beatmap](/wiki/Beatmap) individu. Ini dapat membantu pemain yang mengalami penundaan pendengaran maupun visual. Offset lokal bekerja bersama offset global untuk menghitung offset total.
```
Comment (wowcake) [inline wiki/Glossary/Local_song_offset/id.md:9]:
```suggestion
**Offset lokal** adalah offset yang memindahkan tampilan [hit object](/wiki/Hit_object) dalam kaitannya dengan audio individu dari sebuah [beatmap](/wiki/Beatmap). Ini dapat membantu pemain yang mengalami penundaan pendengaran maupun visual. Offset lokal bekerja bersama offset global untuk menghitung offset total.
```

File: wiki/Glossary/Local_song_offset/id.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Di awal permainan, Offset lokal dapat diubah dengan menekan:
+
+- `+` untuk meningkatkan offset sebesar 5 milidetik
+- `-` untuk mengurangi offset sebesar 5 milidetik
+- `Alt` + `+` untuk meningkatkan offset sebesar 1 milidetik
+- `Alt` + `-` untuk mengurangi offset sebesar 1 milidetik
+
+Jika offset lokal telah ditentukan, osu! akan menampilkan offset lokal pada antarmuka di atas papan skor<!-- TODO: add image of the local offset in the interface -->. osu! juga akan memberi tahu Kamu tentang pengimbangan lokal sebelum memulai permainan.
```
Comment (wowcake) [inline wiki/Glossary/Local_song_offset/id.md:18]:
```suggestion
Jika offset lokal telah ditentukan, osu! akan menampilkan offset lokal pada antarmuka di atas papan skor<!-- TODO: add image of the local offset in the interface -->. osu! juga akan memberi tahu Kamu tentang perubahan lokal offset sebelum memulai permainan.
```

File: wiki/Guides/How_to_Use_the_Offset_Wizard/id.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Bagaimana cara menggunakan Offset Wizard
+
+*Kunjungi juga: [Offset Wizard](/wiki/Options/Offset_Wizard) dan [Universal offset](/wiki/Universal_offset)*
+
+Jika Kamu merasa [hitsound](/wiki/Beatmapping/Hitsound) tidak sinkron dengan musik di setiap beatmap yang dimainkan, kemungkinan besar Kamu perlu mengubah [Universal offset](/wiki/Universal_offset). Cara yang paling mudah untuk melakukan perubahan pada Universal offset adalah dengan menggunakan **Offset Wizard**.
+
+Ikuti langkah-langkah berikut untuk menyesuaikan Universal offset menggunakan Offset Wizard.
+
+## Membukan Offset Wizard
```
Comment (wowcake) [inline wiki/Guides/How_to_Use_the_Offset_Wizard/id.md:9]:
```suggestion
## Membuka Offset Wizard
```

Comment (wowcake) [reply wiki/Guides/How_to_Use_the_Offset_Wizard/id.md:9]: typo

File: wiki/Guides/How_to_Use_the_Offset_Wizard/id.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+1. Buka klien osu!.
+2. Pergi ke menu `Options`, atau tekan kombinasi `Ctrl` + `O` pada keyboard.
+3. Cari/ketik kata "offset" pada pencarian cepat, dan tekan tombol `Offset wizard`.
+
+Kemudian Offset Wizard akan terbuka, memutar lagu yang sebelumnya diputar di klien osu!, sementara bar hijau bergulir di layar dan metronom berdetak.
+
+## Mencocokkan Universal offset
+
+1. Dengarkan lagu dan metronom secara baik-baik ketika dimainkan bersama.
+2. Mulailah menyesuaikan Universal offset ke atas atau bawah, menggunakan tombol panah yang sesuai, atau roda gulir mouse.
```
Comment (wowcake) [inline wiki/Guides/How_to_Use_the_Offset_Wizard/id.md:20]:
```suggestion
2. Mulailah menyesuaikan Universal offset ke atas atau bawah, menggunakan arrow key yang sesuai pada keyboard, atau scroll wheel pada mouse.
```

Comment (wowcake) [reply wiki/Guides/How_to_Use_the_Offset_Wizard/id.md:20]: those terms sound ambiguous

File: wiki/Guides/How_to_Use_the_Offset_Wizard/id.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+   - Jika Kamu merasa hitsound dimainkan terlalu awal dalam game, Kamu perlu *mengurangi* Universal offset.
```
Comment (wowcake) [inline wiki/Guides/How_to_Use_the_Offset_Wizard/id.md:21]:
```suggestion
   - Jika Kamu merasa suara hitsound dimainkan terlalu awal dalam game, Kamu perlu *mengurangi* Universal offset.
```

Comment (wowcake) [reply wiki/Guides/How_to_Use_the_Offset_Wizard/id.md:21]: to make things clearer

File: wiki/Guides/How_to_Use_the_Offset_Wizard/id.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+   - Jika Kamu merasa hitsound dimainkan terlalu cepat dalam game, Kamu perlu *meningkatkan* Universal offset.
```
Comment (wowcake) [inline wiki/Guides/How_to_Use_the_Offset_Wizard/id.md:22]:
```suggestion
   - Jika Kamu merasa suara hitsound dimainkan terlalu cepat dalam game, Kamu perlu *meningkatkan* Universal offset.
```

File: wiki/Guides/How_to_Use_the_Offset_Wizard/id.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+3. Penyesuaian dilanjutkan hingga Kamu mendengar garis metronom menjadi cocok sempurna dengan lagu, menggunakan bar gulir di layar untuk membantu Kamu.
+4. Setelah Kamu berhasil mencocokkan Universal offset, tekan tombol `Back` di kiri bawah layar untuk keluar dari Offset Wizard.
+
+Jika Kamu merasa metronom terasa terlalu cepat atau terlalu lambat untuk lagu tersebut, Kamu mungkin perlu mengaktifkan `Half tick` atau` Double tick` di kiri atas layar untuk mempermudah.
```
Comment (wowcake) [inline wiki/Guides/How_to_Use_the_Offset_Wizard/id.md:26]:
```suggestion
Jika Kamu merasa metronom terasa terlalu cepat atau terlalu lambat untuk lagu tersebut, Kamu mungkin perlu mengaktifkan `Half tick` atau` Double tick` di kiri atas layar untuk mempermudah penyesuaiannya.
```

Comment (wowcake) [reply wiki/Guides/How_to_Use_the_Offset_Wizard/id.md:26]: it sounds empty with just ```mempermudah``` before ending the sentence

File: wiki/Guides/Setting_the_Offset_on_the_Correct_Beat/id.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
+# Bagaiamana cara mengatur offset pada beat yang benar
+
+*[Tentang Offset Interval oleh Alace](https://osu.ppy.sh/community/forums/topics/20998)*
+
+Kebanyakan pemain osu! tidak begitu mahir di musik, jadi kita mempunyai masalah pada beberapa beatmap:
```
Comment (wowcake) [inline wiki/Guides/Setting_the_Offset_on_the_Correct_Beat/id.md:5]:
```suggestion
Kebanyakan pemain osu! tidak begitu mahir dalam hal-hal yang berhubungan dengan musik, akibatnya kita mempunyai masalah pada beberapa beatmap:
```

File: wiki/Guides/Setting_the_Offset_on_the_Correct_Beat/id.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+**Tanda white tick panjang tidak mengikuti awal bagian yang sebenarnya.**
+
+Divisor pertama pada suatu bagian adalah beat yang berat. Lihat contohnya.
+
+***lagu 4/4:***
```
Comment (wowcake) [inline wiki/Guides/Setting_the_Offset_on_the_Correct_Beat/id.md:11]:
```suggestion
***Lagu 4/4:***
```

Comment (wowcake) [reply wiki/Guides/Setting_the_Offset_on_the_Correct_Beat/id.md:11]: i believe this should be capitalized

File: wiki/Guides/Setting_the_Offset_on_the_Correct_Beat/id.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+
+**0** 0 0 0 **0** 0 0 0 **0** 0 0 0 **0** 0 0 0
+
+***lagu 3/4:***
```
Comment (wowcake) [inline wiki/Guides/Setting_the_Offset_on_the_Correct_Beat/id.md:15]:
```suggestion
***Lagu 3/4:***
```

File: wiki/Guides/Setting_the_Offset_on_the_Correct_Beat/id.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+
+**0** 0 0 **0** 0 0 **0** 0 0 **0** 0 0
+
+Dimana **0** adalah beat yang berat, jadi Kamu perlu mengatur offset di padanya.
```
Comment (wowcake) [inline wiki/Guides/Setting_the_Offset_on_the_Correct_Beat/id.md:19]:
```suggestion
Dimana **0** adalah beat yang berat, jadi Kamu perlu mengatur offset di situ.
```

Comment (wowcake) [reply wiki/Guides/Setting_the_Offset_on_the_Correct_Beat/id.md:19]: ```di padanya``` reaaaally doesn't sound right

File: wiki/Options/Offset_Wizard/id.md [hunk 12]
```diff
@@ -0,0 +1,17 @@
+# Offset Wizard
+
+*Kunjungi juga: [Bagaimana cara menggunakan Offset Wizard](/wiki/Guides/How_to_Use_the_Offset_Wizard) dan [Universal offset](/wiki/Universal_offset)*
+
+**Offset Wizard** adalah sebuah alat bawaan yang membantu pengguna dalam mencocokkan [Universal offset](/wiki/Universal_offset) ke nilai yang benar untuk perangkat mereka. Ini diperlukan bila[hitsound](/wiki/Beatmapping/Hitsound) terasa tidak sinkron dengan musik, di setiap beatmap dimainkan. Untuk informasi lebih lanjut tentang cara melakukan ini, kunjungi [Bagaimana cara menggunakan Offset Wizard](/wiki/Guides%2FHow_to_Use_the_Offset_Wizard).
```
Comment (wowcake) [inline wiki/Options/Offset_Wizard/id.md:5]:
```suggestion
**Offset Wizard** adalah sebuah alat bawaan yang membantu pengguna dalam mencocokkan [Universal offset](/wiki/Universal_offset) ke nilai yang benar untuk perangkat mereka. Ini diperlukan bila [hitsound](/wiki/Beatmapping/Hitsound) terasa tidak sinkron dengan musik, di setiap beatmap dimainkan. Untuk informasi lebih lanjut tentang cara melakukan ini, kunjungi [Bagaimana cara menggunakan Offset Wizard](/wiki/Guides%2FHow_to_Use_the_Offset_Wizard).
```

Comment (wowcake) [reply wiki/Options/Offset_Wizard/id.md:5]: you forgot the spacing

File: wiki/Universal_offset/id.md [hunk 13]
```diff
@@ -0,0 +1,13 @@
...
+stub: true
+---
+
+# Universal offset
+
+*Untuk penggunaan lainnya, kunjungi [Offset (disambiguasi)](/wiki/Disambiguation/Offset).*
+
+*Untuk petunjuk langkah demi langkah tentang cara mengatur nilai Universal offset yang benar, lihat [Bagaimana cara menggunakan Offset Wizard](/wiki/Guides/How_to_Use_the_Offset_Wizard).*
+
+**Universal offset** adalah [pengaturan](/wiki/Options) klien osu! yang mempengaruhi timing pada keseluruhan [beatmap](/wiki/Beatmap) di dalam klien osu!. Menyetelnya ke nilai selain `0`, akan menambahkan jeda antara setiap lagu pada sebuah beatmap dan suara grafik yang terkait lainnya. Menyetel Universal offset ke nilai positif akan membuat suara dan grafik yang terkait dengan beatmap muncul lebih awal, sementara menyetelnya ke nilai negatif akan membuatnya muncul kemudian.
```
Comment (wowcake) [inline wiki/Universal_offset/id.md:11]:
```suggestion
**Universal offset** adalah [pengaturan](/wiki/Options) offset audio yang mempengaruhi timing pada keseluruhan [beatmap](/wiki/Beatmap) di dalam klien osu!. Menyetelnya ke nilai selain `0`, akan menambahkan jeda antara setiap lagu pada sebuah beatmap dan suara grafik yang terkait lainnya. Menyetel Universal offset ke nilai positif akan membuat suara dan grafik yang terkait dengan beatmap muncul lebih awal, sementara menyetelnya ke nilai negatif akan membuatnya muncul kemudian.
```

Comment (wowcake) [reply wiki/Universal_offset/id.md:11]: i don't know if this is intentional or not, but if so, can you provide the reasons of why is that?

File: wiki/Universal_offset/id.md [hunk 14]
```diff
@@ -0,0 +1,13 @@
...
+
+Nilai Universal offset yang optimal pada setiap pemain akan berbeda karena variasi dalam sistem mereka. Sebuah Universal offset selain dari `0`, sebaiknya hanya digunakan jika **semua beatmap** mengalami penundaan yang terasa. Jika tidak, [offset lokal](/wiki/Glosarium/Local_song_offset) harus digunakan. Penentuan Universal offset yang salah akan menyebabkan masalah pengaturan timing yang signifikan pada semua beatmap.
```
Comment (wowcake) [inline wiki/Universal_offset/id.md:13]:
```suggestion
Nilai Universal offset yang optimal pada setiap pemain akan berbeda karena variasi dalam sistem mereka. Sebuah Universal offset selain dari `0`, sebaiknya hanya digunakan jika **semua beatmap** mengalami penundaan yang terasa. Jika tidak,  sebaiknya Kamu menggunakan [offset lokal](/wiki/Glosarium/Local_song_offset) saja. Penentuan Universal offset yang salah akan menyebabkan masalah pengaturan timing yang signifikan pada semua beatmap.
```

**Review by wowcake — CHANGES_REQUESTED** [review]

File: wiki/Guides/How_to_Use_the_Offset_Wizard/id.md (hunk 5 above)
Comment (mousesalt) [reply wiki/Guides/How_to_Use_the_Offset_Wizard/id.md:21]: may using without it instead, it's a kind of

File: wiki/Guides/How_to_Use_the_Offset_Wizard/id.md [hunk 15]
```diff
@@ -0,0 +1,26 @@
...
+## Mencocokkan Universal offset
+
+1. Dengarkan lagu dan metronom secara baik-baik ketika dimainkan bersama.
+2. Mulailah menyesuaikan Universal offset ke atas atau bawah, menggunakan arrow key yang sesuai pada keyboard, atau scroll wheel pada mouse.
+   - Jika Kamu merasa hitsound dimainkan terlalu awal dalam game, Kamu perlu *mengurangi* Universal offset.
+   - Jika Kamu merasa hitsound dimainkan terlalu cepat dalam game, Kamu perlu *meningkatkan* Universal offset.
+3. Penyesuaian dilanjutkan hingga Kamu mendengar garis metronom menjadi cocok sempurna dengan lagu, menggunakan bar gulir di layar untuk membantu Kamu.
+4. Setelah Kamu berhasil mencocokkan Universal offset, tekan tombol `Back` di kiri bawah layar untuk keluar dari Offset Wizard.
+
+Jika Kamu merasa metronom terasa terlalu cepat atau terlalu lambat untuk lagu tersebut, Kamu mungkin perlu mengaktifkan `Half tick` atau `Double tick` di kiri atas layar untuk mempermudah penyesuaiannya.
```
Comment (Fhoxydeo) [inline wiki/Guides/How_to_Use_the_Offset_Wizard/id.md:26]: too much conjunction between `merasa` and `terasa`

Comment (wowcake) [reply wiki/Guides/How_to_Use_the_Offset_Wizard/id.md:26]: i think it is good enough and don't need any changing

**Review by wowcake — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

File: wiki/Guides/How_to_Use_the_Offset_Wizard/id.md (hunk 15 above)
Comment (TicClick) [reply wiki/Guides/How_to_Use_the_Offset_Wizard/id.md:26]: if this is not worth fixing, call me for the merge

**Pull request merged**